.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,30 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue related to pipelines
```

Comment:
```suggestion
  - name: Issues related to Kubeflow Pipelines
```

Reply:
Should we add `Kubeflow` as prefix of components? I think this might be a little bit redundant. 

How about 

```
name: Issues related to Pipelines
```